components/breadcrumbs.js

```diff
@@ -25,21 +26,36 @@ const separator = css`
 
 const urlStyle = css`
   color: #6d6363;
+  text-decoration: underline;
+  font-size: 16px;
 `;
 
 const iconStyle = css`
   vertical-align: bottom;
 `;
 
-export class Breadcrumbs extends Component {
+export class BreadCrumbs extends Component {
   render() {
+    const { breadcrumbs } = this.props;
     return (
-      <div className={whiteBanner}>
+      <div className={greyBanner}>
         <Container>
           <div className={breadCrumbStyle}>
-            <a href={this.props.url} className={urlStyle}>
-              <HomeIcon className={iconStyle} /> Home
-            </a>
+            <HeaderLink id="homeButton" href={"/"} className={urlStyle}>
```

Comment:
One more bug: if you're on /benefits-directory, you make some selections, then click the Home breadcrumb, the url params are cleared and you stay on the same page without going to home. 

I think this would be fixed with `href={mutateUrl("")}`